doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -523,7 +523,7 @@ cells:
 
 
 - code: |
-    read_hdf(filepath_hdf, key='deaths', sort_rows=True)
+    read_hdf(filepath_hdf, key='deaths').sort_axes()
```

Comment:
.sort_axes should not be necessary. Either the data is sorted in the file and the method does not do anything, or it is not sorted and it should be.

Reply:
hmmm, nevermind! I didn't read the context, **sorry**! But since read_hdf is different from the other methods, maybe we should add a comment to make users notice the difference?